test/CpmsClientTest/MockApiService.php

```diff
@@ -9,34 +11,28 @@
  *
  * @package CpmsClientTest
  */
-class MockApiService extends ApiService
+#[AllowDynamicProperties] class MockApiService extends ApiService
```

Comment:
i am not a fan of using "#[AllowDynamicProperties]". Usually it comes up to declaring class variables for those using class scope like: private $var, when in code we declare $this->var...

Reply:
removed